maas/client/viscera/machines.py

```diff
@@ -174,6 +177,8 @@ class Machine(Object, metaclass=MachineType):
         :param hwe_kernel: The HWE kernel to deploy. Probably only relevant
             when deploying Ubuntu.
         :param comment: A comment for the event log.
+        :param wait: If specified, wait until the deploy is complete, polling
+            at the specified frequency.
```

Comment:
Well, having wait=True separately seems a little redundant. Maybe the issue is really just that 'wait' is not clear, so it could be:

machine.deploy(wait_interval=5)